c_tests/run.rs

```diff
@@ -37,6 +37,10 @@ fn mk_compiler(exe: &Path, src: &Path, opt: &str) -> Command {
         "-flto",
         "-Wl,--mllvm=--embed-bitcode-final",
         "-Wl,--lto-basic-block-sections=labels",
+        // A trace may reference globals that weren't exported by the linker. This can make it
+        // impossible to find the global's address at runtime and update the JITModule's mapping.
+        // Using `--export-dynamic` enforces that all globals are exported, allowing the compiled
```

Comment:
s/enforces that all globals are/forces all globals to be/

Also we should probably use the "AOT IR" / "JIT IR" terminology to avoid confusing our future selves :)

Reply:
Fixed in 1684bec